Telemetry/TelemetryStorage.swift

```diff
@@ -11,31 +11,20 @@ import Foundation
 public class TelemetryStorage {
     private let name: String
     private let configuration: TelemetryConfiguration
-    
+
+    let userDefaults = UserDefaults(suiteName: "org.mozilla.telemetry-ios-lib")!
```

Comment:
Sorry about the unwrap, this constructor only fails if one tries to use a suitename that clashes with the app suite or the global namespace. AFAIK it is considered safe to force-unwrap in other cases.

Reply:
I think this needs to be app-specific though. For instance, wouldn't this cause data to overlap between Focus and Firefox on the same device?

We can probably do something similar to how we get the app's `UserDefaults` already in the `UserDefaultsMeasurement` here: https://github.com/mozilla-mobile/telemetry-ios/blob/master/Telemetry/TelemetryMeasurements.swift#L376